### PR TITLE
README.md updates

### PR DESCRIPTION
**Pull request by belagertem**
Updated section _Which Manipulation is required for which test_

# Checklist

The following aspects have been respected by the author of this pull request, confirmed by both pull request assignee **and** reviewer:

* Adherence to coding conventions
  * [x] Pull Request Assignee
  * [x] Reviewer
* Adherence to javadoc conventions
  * [x] Pull Request Assignee
  * [x] Reviewer
* Changelog update (necessity checked and entry added or not added respectively)
  * [x] Pull Request Assignee
  * [x] Reviewer
* README update (necessity checked and entry added or not added respectively)
  * [x] Pull Request Assignee
  * [x] Reviewer
* config update (necessity checked and entry added or not added respectively)
  * [x] Pull Request Assignee
  * [x] Reviewer
* SDCcc executable ran against a test device (if necessary)
  * [x] Pull Request Assignee
  * [x] Reviewer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -174,7 +174,7 @@ The test tool has the following limitations. If the DUT falls under these limita
 | **Requirement** | **T2IAPI Manipulation**                                                              | 
 |-----------------|--------------------------------------------------------------------------------------|
 | R0025_0         | SetComponentActivation                                                               |
-| R0029           | SetAlertConditionPresence                                                            |
+| R0029           | SetAlertActivation, SetAlertConditionPresence                                        |
```
Comment (ben-Draeger) [inline README.md:177]: Sollte das Requirement nicht R0029_0 heissen?

File: README.md [hunk 2]
```diff
@@ -185,15 +185,15 @@ The test tool has the following limitations. If the DUT falls under these limita
 | R0124           | CreateContextStateWithAssociation                                                    |
 | R0125           | CreateContextStateWithAssociation                                                    |
 | R0133           | CreateContextStateWithAssociation                                                    |
-| B-61            | SetMetricQualityValidity                                                             |
```
Comment (ben-Draeger) [inline README.md:188]: Den Test scheint es noch zu geben und er scheint auch noch die Manipulation SetMetricQualityValidity zu verwenden. Warum sollte der Eintrag in der Tabelle entfernt werden?

**Comment by ben-Draeger** [comment]
The GetRemovableDescriptors manipulation was renamed to GetRemovableDescriptorsOfClass.
When updating the table, we should change that as well.

File: README.md (hunk 1 above)
Comment (belagertem) [reply README.md]: indeed, changed it

**Comment by belagertem** [comment]
> The GetRemovableDescriptors manipulation was renamed to GetRemovableDescriptorsOfClass. When updating the table, we should change that as well.

done

File: README.md (hunk 2 above)
Comment (belagertem) [reply README.md:188]: You removed this test a few hours ago (see https://github.com/Draegerwerk/SDCcc/pull/77)

Comment (ben-Draeger) [reply README.md:188]: Indeed. This change had not yet reached my main. I had to sync my fork to see it.

**Pull request merged**